src/components/Dashboard/ActionRequired.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-expansion-panels v-model="panel">
+    <v-expansion-panel class="action-required-container">
+      <v-card flat class="px-6 py-5">
+        <h3>Action Required</h3>
+        <p class="mt-1">
+          BC Registries is missing information about your business (e.g.. business start date, nature of business,
+          business address etc.). Please contact BC Registries to input any missing business information. Missing
+          information must be entered before toy can file changes or dissolve this business.
+          <v-btn class="details-btn" color="primary" text :ripple="false" @click.stop="togglePanel">
+            <v-icon left color="primary">mdi-information-outline</v-icon>
+            <span color="primary">{{ panel === 0 ? "Hide Details" : "View Details" }}</span>
+          </v-btn>
+        </p>
+        <v-expansion-panel-content class="action-required-contact-info">
+          <p>BC Registries Contact Information:</p>
+          <ContactInfo class="mt-4" />
+        </v-expansion-panel-content>
+      </v-card>
+    </v-expansion-panel>
+  </v-expansion-panels>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { ContactInfo } from '@/components/common'
+
+@Component({
+  components: {
+    ContactInfo
+  }
+})
+export default class LegalObligation extends Vue {
+  public panel: number = 1;
+
+  public togglePanel (): void {
+    this.panel = this.panel === 1 ? 0 : 1
+  }
+}
+</script>
+
+<style lang="scss">
```

Comment:
Need to remove global padding v-expansion-panel-content__wrap which come in contact section